inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt

```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
+jstyleson==0.0.2
+numpy==1.16.3
+addict==2.2.1
+pandas==0.24.2
+hyperopt==0.1.2
+networkx==2.2
+tqdm==4.31.1
+py-cpuinfo==4.0.0
```

Comment:
does it planed to be used with DL Workbench?
If I right remember, they have py-cpuinfo==7.0.0 as requirements

Reply:
This actually comes from AC, POT does not use it directly as far as I can see). So again, the task is to gather all the requirements across OV components, find misalignments, find the common denominator for each dependency (probably the highest version), fix versions, and let teams validate them.

Reply:
in AC this component has py-cpuinfo!=5.0,!=6.0.0 (these 2 version broken for working in frozen environment)
My question more about, is there is plans to migrate DL Workbench docker on pypi dev package usage or not. For us current version is ok, there is no conflict between components inside dev-package.

Not sure that we need the whole requirements alignment across all OV components in wheel creation scope, this task is wider than represented tools.

Reply:
updated to py-cpuinfo!=5.0,!=6.0

Reply:
We just added "texttable==1.6.3" in POT. POT also requires a torch cpu version. It is not installed by default but it is requested to be installed if the user is going to use specific optimization methods.

Reply:
Added texttable==1.6.3, but how do you suggest handling the torch cpu version?

Reply:
As per our discussion, let keep torch out of the sope for now.